env/set-client-id.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs"); // eslint-disable-line
+
+const MY_NOTES_CLIENT_ID_DEFAULT = "813847979218-unbbh8m0el8vjdi74rnqdarm1mhv3985.apps.googleusercontent.com";
+const clientId = process.env.MY_NOTES_CLIENT_ID || MY_NOTES_CLIENT_ID_DEFAULT;
+
+const manifestPath = process.argv.slice(2)[0];
+const manifestContent = JSON.parse(fs.readFileSync(manifestPath, "utf8"));
+manifestContent.oauth2.client_id = clientId;
+fs.writeFileSync(manifestPath, JSON.stringify(manifestContent, null, 2));
```

Comment:
**Before:**
Client ID is hard-coded into `manifest.json`.

**Now:**
Client ID can be easily modified by setting a `MY_NOTES_CLIENT_ID` enviroment variable. If not set, a default is used.

---

What is Client ID?
> Client ID is needed for Google Drive Sync to work. It is bound to the extension ID, meaning other extension cannot use it if it wasn't created for it.
>
> Client ID is "OAuth 2.0 Client ID" and is created in Google Cloud Platform under "APIs and Services". It is used to identify the extension, so it can then ask for access token to access Google Drive.